src/constants/theme/colors.ts

```diff
@@ -1,29 +1,29 @@
 export const COLORS = {
```

Comment:
- ![primary](https://via.placeholder.com/15/000e38/000000?text=+) `primary`
- ![white](https://via.placeholder.com/15/ffffff/000000?text=+) `white`
- ![background](https://via.placeholder.com/15/f9fafb/000000?text=+) `background`
- ![textColor](https://via.placeholder.com/15/3a4049/000000?text=+) `textColor`
- ![textSecondaryColor](https://via.placeholder.com/15/5b616a/000000?text=+) `textSecondaryColor`
- ![placeholder](https://via.placeholder.com/15/777e91/000000?text=+) `placeholder` also used as button background when selected. Not used for individual area placeholder
- ![disabled](https://via.placeholder.com/15/e0e4ed/000000?text=+) `disabled`
- ![secondarySelectedColor](https://via.placeholder.com/15/f57f7a/000000?text=+) `secondarySelectedColor` color of the chevron when list item is pressed
- ![exerciseProgressIndicator](https://via.placeholder.com/15/20c5b4/000000?text=+) `exerciseProgressIndicator`
- ![containedButtonSelected](https://via.placeholder.com/15/3d4662/000000?text=+) `containedButtonSelected`
- ![overlay](https://via.placeholder.com/15/000e38/000000?text=+) `overlay` +80% transparency
- ![functionalCorrect](https://via.placeholder.com/15/0adb75/000000?text=+) `functionalCorrect`
- ![functionalAlmostCorrect](https://via.placeholder.com/15/ffbb4a/000000?text=+) `functionalAlmostCorrect`
- ![functionalIncorrect](https://via.placeholder.com/15/ff5252/000000?text=+) `functionalIncorrect`
- ![audioIconHighlight](https://via.placeholder.com/15/F1635F/000000?text=+) `audioIconHighlight`
- ![audioIconSelected](https://via.placeholder.com/15/ffa3a3/000000?text=+) `audioIconSelected`
- ![shadow](https://via.placeholder.com/15/000e38/000000?text=+) `shadow` + 60% transparency
- ![lunesArtikelDas](https://via.placeholder.com/15/72f399/000000?text=+) `lunesArtikelDas`
- ![lunesArtikelDiePlural](https://via.placeholder.com/15/eee12d/000000?text=+) `lunesArtikelDiePlural`
- ![lunesArtikelDie](https://via.placeholder.com/15/faa7a7/000000?text=+) `lunesArtikelDie`
- ![lunesArtikelDer](https://via.placeholder.com/15/8cc8f3/000000?text=+) `lunesArtikelDer`
- ![blue](https://via.placeholder.com/15/0000FF/000000?text=+) `blue`
- ![lunesBlueLight](https://via.placeholder.com/15/97aef1/000000?text=+) `lunesBlueLight`
- ![lunesOrangeLight](https://via.placeholder.com/15/ffbb4a/000000?text=+) `lunesOrangeLight`
- ![lunesGreenLight](https://via.placeholder.com/15/7ae6d8/000000?text=+) `lunesGreenLight`

Reply:
hmh, naming is still not really good. Sometimes it deplends on how it is used (disabled, audioIconSelected...) and sometimes it depends on what it looks linke (blue).
`Primary` is for text? so maybe naming it textColorPrimary would make more sense?
Also the functional-prefix does not really make sense in my opinion.

Also it would be nice, if the naming has a consistent pattern, for example we have: textSecondaryColor and secondarySelectedColor. Once we have the `secondary` as the first part and once as the second part.
And some colors have the suffix -color and some dont.

Reply:
I adjusted the names. However I feel like some colors should be reevaluated, as e.g. secondarySelected or when do we use white and when do we use the background color. Or sometimes we use a different color for placeholders (exercise and customDiscipline). I'd link this in a lunes typography issue, but currently there's only one for integreat. Is it okay to create an issue, or would that be Ulrikes field of work?

Reply:
I think it is fine if you open a ticket or even adjust it right now, as you prefer.

Reply:
then i'll do that immediately